t/api/03-auth.t

```diff
@@ -161,8 +161,8 @@ subtest 'wrong api key - replay attack' => sub() {
                 'X-API-Hash'      => hmac_sha1_sum($self->_path_query($tx) . $timestamp, $self->apisecret),
             );
 
-            while (my ($k, $v) = each %headers) {
-                $tx->req->headers->header($k, $v);
+            foreach my $key (sort keys %headers) {
```

Comment:
The sorting here brings likely no benefit.

Reply:
I was thinking it would affect error output with invalid values... but that API checks the headers in a specific order anyway so it indeed makes no difference.